make/modules/java.base/Lib.gmk

```diff
@@ -233,3 +233,23 @@ ifeq ($(ENABLE_FALLBACK_LINKER), true)
 
   TARGETS += $(BUILD_LIBFALLBACKLINKER)
 endif
+
+################################################################################
+
+ifeq ($(call isTargetOs, linux)+$(call isTargetCpu, x86_64)+$(INCLUDE_COMPILER2), true+true+true)
+  $(eval $(call SetupJdkLibrary, BUILD_LIB_X86_64, \
```

Comment:
As this is a C++ lib, consider using g++ for linking by setting:
```
TOOLCHAIN := TOOLCHAIN_LINK_CXX
```

Reply:
Please see the linking updated to use g++ in the latest commit.